src/test/java/teammates/test/cases/logic/CoursesLogicTest.java

```diff
@@ -438,12 +438,24 @@ private void testGetCoursesForStudentAccount() throws Exception {
         assertEquals(2, courseList.size());
 
         CourseAttributes course1 = dataBundle.courses.get("typicalCourse1");
-        assertEquals(course1.getId(), courseList.get(0).getId());
-        assertEquals(course1.getName(), courseList.get(0).getName());
+        int temp = 0;
+        for (CourseAttributes course : courseList) {
+            if (course.compareTo(course1) == 0) {
```

Comment:
@wkurniawan07  I don't know why but `equals` is not working here but `compareTo` is working fine.